pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -193,201 +193,243 @@ export default function HandlerForm({id, endpointId}) {
+                            data={handler && handler.sequence && handler.sequence}
+                            nameOverride={"Sequence"} required/>
+                        </div>
+                        <div className="col-6">
+                          <GenericInputComponent
+                            type={"text"}
+                            name={"template"}
+                            id={"templateInput"}
+                            data={handler && handler.template && handler.template}
+                            nameOverride={"Template"}
+                          />
+                        </div>
+                      </div>
+                      <br/>
+                      <div className="row">
+                        <div className="col-6">
+                          {
+                            entities !== null && entities.length > 0 ? (
+                              <div className="form-group">
+                                {handler !== null &&
+                                handler.entity !== undefined &&
+                                handler.entity !== null ? (
+                                    <SelectInputComponent
+                                      options={entities}
+                                      data={handler.entity}
+                                      name={"entity"} id={"entityInput"}
+                                      nameOverride={"Entity"}
+                                      value={"/admin/entities/"}/>
+                                  )
+                                  : (
+                                    <SelectInputComponent
+                                      options={entities}
+                                      name={"entity"} id={"entityInput"}
+                                      nameOverride={"Entity"}
+                                      value={"/admin/entities/"}/>
+                                  )}
+                              </div>
+                            ) : (
+                              <SelectInputComponent
+                                options={[]}
+                                name={"entity"} id={"entityInput"}
+                                nameOverride={"Entity"}
+                                value={"/admin/entities/"}/>
+                            )
+                          }
+                        </div>
+                        <div className="col-6">
+                          {handler !== null && handler.type !== null ? (
+                            <SelectInputComponent
+                              options={[
+                                {name: "twig", value: "twig"},
+                                {name: "markdown", value: "markdown"},
+                                {name: "restructuredText", value: "restructuredText"},
+                              ]}
+                              name={"templateType"}
+                              id={"templateTypeInput"}
```

Comment:
On line `138` you're selecting `even.target.templateType.value` and not `event.target.templateTypeInput`. This selector is based on this `id` you're setting.

So you're setting `templateTypeInput`, but trying to retrieve the data from `templateType` which results in an error (see screenshot).

![image](https://user-images.githubusercontent.com/66552932/152744536-3eef87df-489e-41a1-9296-e12ddf651791.png)